class Failure {
  final String msg;
  Failure([this.msg = "An unexpected error occurred!"]);
}
